docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -0,0 +1,585 @@
+:linkTroubleshooting: https://github.com/integr8ly/example-customisations/blob/master/docs/troubleshooting.adoc
+:linkGitHubFork: https://help.github.com/articles/fork-a-repo/
+:linkIndexURL: https://github.com/integr8ly/example-customisations/index.adoc
+:walkthrough-1A: *Integrating message-oriented middleware with a RESTful API using AMQ Online*
+:walkthrough-2: *Managing and protecting APIs using API management*
+:publishing-walkthrough: *Adding walkthroughs to a cluster*
+
+= Writing {PRODUCT} Walkthroughs
+
+
+== About Walkthroughs
+
+
+Walkthroughs are tutorials available in {PRODUCT}. The default walkthroughs guide you through the following scenarios:
+
+* {walkthrough-1A}
+* {walkthrough-2}
+
+These walkthroughs are available in the {web-app}.
```

Comment:
By these you mean the default ones?

Reply:
right, I need to add 'default'. We're still debating how to label them in web app, currently we're thinking these are 'Red Hat' walkthroughs, and all others are 'Community' walkthroughs